### PR TITLE
python 3.6 changes

### PR DESCRIPTION
**Pull request by yzainee**

### CODE REVIEW

File: runtests.sh [hunk 1]
```diff
@@ -54,13 +54,13 @@ function setup_virtualenv {
...
-    pip3 install -r requirements.txt
+    pip3.6 install -r requirements.txt
 
     # Install profiling module
-    pip3 install pytest-profiling
+    pip3.6 install pytest-profiling
 
     # Install pytest-coverage module
-    pip3 install pytest-cov
+    pip3.6 install pytest-cov
```
Comment (abs51295) [inline runtests.sh]: use `pip3` instead of `pip 3.6`

**Comment by yzainee** [comment]
[test]

**Comment by msehnout** [comment]
You need to install OpenSSL header files from the `openssl-devel` package.

**Comment by tisnik** [comment]
please squash commits before merge

**Comment by yzainee** [comment]
Yes, i will squash the commits. But presently, its hasnt yet passed :(

**Comment by tisnik** [comment]
LGTM @yzainee , please squash+merge, we all are done :)

**Comment by yzainee** [comment]
[test]

**Comment by yzainee** [comment]
[test]

**Comment by tisnik** [comment]
[test]

**Comment by yzainee** [comment]
[test]

**Comment by tisnik** [comment]
[test]

**Pull request merged**